tbtc-ts/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@keep-network/tbtc-v2-ts",
+  "version": "0.1.0",
+  "license": "MIT",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "scripts": {
+    "format": "yarn run lint && prettier --check .",
+    "format:fix": "yarn run lint:fix && prettier --write .",
+    "lint": "eslint . --ext .js,.ts",
+    "lint:fix": "eslint . --ext .js,.ts --fix",
+    "test": "mocha --exit --recursive 'test/**/*.test.ts'",
```

Comment:
I created a test folder and a simple test and it didn't work:
```
import { expect } from "chai"

describe("Test", async () => {
  it("test", async () => {
    expect(1).to.equal(1)
  })
})
```

The error was: `SyntaxError: Cannot use import statement outside a module`.

Replacing
```
"test": "mocha --exit --recursive 'test/**/*.test.ts'"
```
with
```
"test": "mocha --require ts-node/register --exit --recursive 'test/**/*.test.ts'",
```
seems to fix the issue

Reply:
Good catch! Fixed it in https://github.com/keep-network/tbtc-v2/pull/80/commits/efb3087e408bfc207cf450e5a6ae95557605dab2 by adding an appropriate `.mocharc.json` file.